Drop-In/src/main/res/layout/bt_fragment_add_card.xml

```diff
@@ -1,52 +1,39 @@
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:bt="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:orientation="vertical"
+    android:background="@color/bt_base_background"
     tools:context="com.braintreepayments.api.AddCardFragment">
 
-    <androidx.appcompat.widget.Toolbar
-        android:id="@+id/bt_toolbar"
```

Comment:
I think we need this toolbar right? I'm getting an error in the demo app from AddCardFragment with this missing.

Reply:
Yeah this was a rebase error i'm going to revert.

Reply:
Great catch! It should be good now.